public/locales/en-US/send.ftl

```diff
@@ -117,13 +117,10 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
-#
-#
-#
-# shared ->
-#
-#
-#
+
+# Denotes strings shared across versions
+ [[ shared ]]
+
 #  title = Firefox Send
```

Comment:
There are still commented out strings. What the outcome you hope to get by doing this? Right now this will be considered a single huge comment for the next string `send-brand`, and displayed accordingly in Pontoon.

By commenting them, you'll still lose all translations. Either you keep them uncommented, or you remove them.

Reply:
so i talked to @dannycoates these are strings that don't need to be localized b/c they already are. however since they show up as new in the github diff we didn't want to cause confusion. I've spoken with danny and we'll merge my PR into his branch as is and remove the comments in his branch

Reply:
I'll move this comment over to danny's PR so we can merge this one and resolve there

Reply:
You should remove them completely, or leave them as they are (uncommented). Commenting them out is the only option that doesn't work.

Reply:
OK, I think I got confused by the comment. You plan to leave them as they are (remove the comments).

Reply:
yes